frontend/src/app/modules/global_search/global-search-work-packages.component.ts

```diff
@@ -108,13 +100,13 @@ export class GlobalSearchWorkPackagesComponent extends WorkPackageEmbeddedTableC
       this.globalSearchService.projectScope$
     )
     .pipe(
+      skip(1),
       distinctUntilChanged(),
       debounceTime(10),
       untilComponentDestroyed(this)
     )
     .subscribe(([newSearchTerm, newProjectScope]) => {
       this.WpFilter.visible = false;
-      this.firstTimeLoadForm = true;
```

Comment:
@ulferts If we remove this line, we run into trouble. Say, you search, open "Advanced Filters" then the searched term shall be set as value of the SearchFilter. Then you trigger a new search through the GlobalSearchInput and open "Advanced filters" again. If we remove this line, the value of the SearchFilter does not update, and that is bad. Obviously, loading the form everytime we perform a new search isn't optimal either but at least "it works".

Reply:
Will take a look at it, @wielinde. Thanks for finding.